[Lighthouse] Style and link updates (#4503)
GoogleChrome/developer.chrome.com: site/en/docs/lighthouse/best-practices/image-aspect-ratio/index.md

---

If a rendered image has an [aspect ratio][ar] that's significantly different
to the aspect ratio in its source file (the _natural_ aspect ratio),
the rendered image may look distorted,
possibly creating an unpleasant user experience.

## How the Lighthouse image aspect ratio audit fails

[Lighthouse](/docs/lighthouse/overview/) flags any

> image with a rendered dimension of more than a few pixels difference to the expected dimension

Author: Defer to others on whether I'm actually right for this one or not

There are two common causes for an incorrect image aspect ratio:

> - An image is set with explicit width and height values that differ from the source image's dimensions.

Author: ```suggestion
- An image is set to explicit width and height dimensions that differ from the source image's dimensions.
```

Alternately (keeps the 'to')
